efi/efi.go

```diff
@@ -183,3 +183,21 @@ func WriteEFIVariable(variable string, buf []byte) error {
 	}
 	return nil
 }
+
+// Return the boot entry which is currently booted.
+func GetCurrentlyBootedEntry() (string, error) {
+	_, data, err := attributes.ReadEfivarsWithGuid(
+		"LoaderEntrySelected",
+		*util.StringToGUID("4a67b082-0a4c-41cf-b6c7-440b29bb8c4f"),
```

Comment:
It'd be nice to confirm if gummiboot uses the same var.

Reply:
It does :)
https://github.com/karelzak/gummiboot/blob/8ca98d57fff8b5fa61c01f18eeb103aac2d1591d/src/efi/gummiboot.c#L1974

Reply:
Nice, thanks!